File: fabric/src/main/java/dev/niostone/interstellar/fabric/InterstellarFabric.java
package dev.niostone.interstellar.fabric;

import dev.niostone.interstellar.Interstellar;
import net.fabricmc.api.ModInitializer;

public class InterstellarFabric implements ModInitializer {
    @Override
    public void onInitialize() {
        Interstellar.init();
    }
}

File: fabric/src/main/kotlin/dev/niostone/interstellar/fabric/InterstellarFabric.kt
package dev.niostone.interstellar.fabric

import dev.niostone.interstellar.impl.Interstellar
import net.fabricmc.api.ModInitializer

object InterstellarFabric: ModInitializer {
    override fun onInitialize() {
        Interstellar.init()
    }
}